plasmapy/physics/distribution.py

```diff
@@ -58,35 +63,460 @@ def Maxwellian_1D(v: units.m/units.s,
     mass m, velocity v, a drift velocity V and with temperature T is:
 
     .. math::
-        f = \sqrt{\frac{m}{2 \pi k_B T}} e^{-\frac{m}{2 k_B T} (v-V)^2}
+    	f = \sqrt{\frac{m}{2 \pi k_B T}} e^{-\frac{m}{2 k_B T} (v-V)^2}
+    	f = (\pi * v_Th^2)^{-1/2} e^{-(v - v_{drift})^2 / v_Th^2}
+    	where v_Th = \sqrt(2 k_B T / m) is the thermal speed
 
     Examples
     --------
     >>> from plasmapy.physics import Maxwellian_1D
     >>> from astropy import units as u
     >>> v=1*u.m/u.s
     >>> Maxwellian_1D(v=v, T= 30000*u.K, particle='e',V_drift=0*u.m/u.s)
-    <Quantity 5.916329687405701e-07 s / m>
+    <Quantity 5.916329687405703e-07 s / m>
+    """
+    # convert temperature to Kelvins
+    T = T.to(u.K, equivalencies=u.temperature_energy())
+    # get thermal velocity and thermal velocity squared
+    vTh = thermal_speed(T, particle=particle, method="most_probable")
+    vThSq = vTh ** 2
+    # Get square of relative particle velocity
+    vSq = (v - V_drift) ** 2
+    # calculating distribution function
+    try:
+        coeff = (vThSq * np.pi) ** (-1 / 2)
+        expTerm = np.exp(-vSq / vThSq)
+        distFunc = coeff * expTerm
+    except Exception:
```

Comment:
Code coverage whines about this one, so either add some test with `pytest.raises(ValueError)`  or simply plot down a `# coveralls: ignore` after the `Exception:` to skip testing this one!

Reply:
Or just remove the try-except block? I don't think it's doing much here.

Reply:
We could do that as well. I'm not 100% sure why they pop up everywhere.

Reply:
I just started by imitating other code I've seen in this package, but it seems like these try-except blocks are mostly useless for such simple functions.